### PR TITLE
Use tools v6.1, improve .env docs

### PR DESCRIPTION
**Pull request by nyurik**
* Add info about using MID_ZOOM to optimize tile generation
* Remove params for generation that are now automatically detected

### CODE REVIEW

**Review by Queens666 — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -18,14 +18,22 @@ PGPORT=5432
...
+# If MID_ZOOM is 11, it will avoid generating z12..z14 = 4+16+64 = 84 tiles for each empty z11 tile.
+# This setting only works with `make generate-tiles-pg` command.  Make sure MID_ZOOM < MAX_ZOOM.
+# MID_ZOOM=11
+
 # Use  true  (case sensitive) to allow data updates
 DIFF_MODE=false
 
 # Hide some output from Mapnik tile generation for clarity
+# This setting is only used with the legacy  `make generate-tiles` command.
 FILTER_MAPNIK_OUTPUT=1
```
Comment (TomPohys) [inline .env:38]: `FILTER_MAPNIK_OUTPUT=1` was removed in https://github.com/openmaptiles/openmaptiles-tools/pull/349, this should be removed too.

```suggestion

```

File: .env [hunk 2]
```diff
@@ -18,14 +18,22 @@ PGPORT=5432
...
-# Which zooms to generate in   make generate-tiles
+# Which zooms to generate with   make generate-tiles-pg
 MIN_ZOOM=0
 MAX_ZOOM=7
 
+# Speed up tile generation: when MID_ZOOM is set, generate all tiles MIN..MID,
+# and only generate MID+1..MAX if prior zoom is not empty.
+# This way empty tiles with water or land do not regenerate for all zooms after MID.
+# If MID_ZOOM is 11, it will avoid generating z12..z14 = 4+16+64 = 84 tiles for each empty z11 tile.
+# This setting only works with `make generate-tiles-pg` command.  Make sure MID_ZOOM < MAX_ZOOM.
```
Comment (TomPohys) [inline .env:29]: Explanation of this function was done in https://github.com/openmaptiles/openmaptiles-tools/pull/383. Could be shortened to e.g. in suggestion? It keeps the `.env` file short :-) 

```suggestion
# `MID_ZOOM` setting only works with `make generate-tiles-pg` command.  Make sure MID_ZOOM < MAX_ZOOM.
# See https://github.com/openmaptiles/openmaptiles-tools/pull/383
```

**Comment by TomPohys** [comment]
Hi @nyurik, is there any blocker for this? Or could I merge it?

**Comment by TomPohys** [comment]
I commit my suggestions for finishing the preparation for release v3.13. Thank you for your PR.

**Comment by nyurik** [comment]
all good, thx for merging.  We may want to improve links/docs about MID_ZOOM, and eventually also rename `generate-tiles-pg` into the default `generate-tiles`. I'm hoping we can drop mapnik by v4

**Pull request merged**